spec/controllers/samples_reports_controller_spec.rb

```diff
@@ -9,6 +9,14 @@
     @other_institution = Institution.make! user: @other_user
 
     @box = Box.make!(:overfilled, institution: @institution, purpose: "LOD")
+
+
+    concentrations = @box.samples.map(&:concentration)
+    signals = Numo::DFloat[*@box.samples.map(&:measured_signal)]
+
+    puts concentrations.inspect
+    puts signals.inspect
```

Comment:
manual debug leftovers

Reply:
@ysbaddaden ah zut. Should I do a revert or a new branch?

Reply:
No need for a revert, we'd end up with 3 commits in total (old commit, revert commit, new commit) :smile:

Create another branch and squash it so we'll have 2 commits (commit, fixup commit)